packages/odata-vocabularies/src/vocabulary-service.ts

```diff
@@ -437,9 +453,37 @@ export class VocabularyService {
             values.unshift(...enumTypeDocumentation);
         }
 
+        if (
+            (element.kind === COMPLEX_TYPE_KIND || element.kind === PROPERTY_KIND || element.kind === TERM_KIND) &&
+            element.constraints?.applicableTerms
```

Comment:
Are we sure that applicableTerms is not empty array?

Reply:
I think empty array just disables embedded annotations. Our code should work properly even with empty array

Reply:
I don't think that an empty array occurs in vocabularies - i.e. that it is a desired use case (but not sure). I was more concerned about that it could happen internally (defaulting) and will lead to undesired output in the generated documentation.
Hence I would rather check for `element.constraints?.applicableTerms?.length`

Reply:
Ah... I see your point. In this specific place it makes sense. But imho if array is empty we still should show in description some text, a-la 'No applicable terms' or something. But since we don't have any realistic use case, we can go with current implementation. We never create empty array internally. Empty constraints object may occur, but empty specific constraint should not. Otherwise it would be a bug